_posts/2016-01-14-javascript-code-coverage-with-instanbul-and-coveralls.markdown

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: "JavaScript Code Coverage With Instanbul and Coveralls"
+date: 2016-01-14T21:50:18-02:00
+keywords: test,
+author: rafaeleyng
+excerpt: >
+  A fast lane to set up code coverage in your project
```

Comment:
I think it would be nice to provide a little more context of why someone should read this post. Maybe indicate that this will show how to setup code coverage in javascript projects, integrating with the CI, .... it's up to you, just a suggestion.